src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,76 @@
-export const Pagination = () => {};
+import React from 'react';
+import cn from 'classnames';
+import { Link } from 'react-router-dom';
+import { getNumbers } from '../../utils';
+
+interface Props {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+  itemsOnPage: string[];
```

Comment:
The 'itemsOnPage' prop doesn't seem to be used for rendering the pagination directly. Instead, it's used to determine if the 'next' button should be disabled. Consider renaming this prop to reflect its purpose more accurately, such as 'isLastPage' which could be a boolean.